core/src/main/scala/org/apache/spark/deploy/worker/ui/WorkerPage.scala

```diff
@@ -112,7 +112,15 @@ private[ui] class WorkerPage(parent: WorkerWebUI) extends WebUIPage("") {
       <td>
         <ul class="unstyled">
           <li><strong>ID:</strong> {executor.appId}</li>
-          <li><strong>Name:</strong> {executor.appDesc.name}</li>
+          <li><strong>Name:</strong>
+          {
+            if ({executor.state == ExecutorState.RUNNING}) {
+              <a href={executor.appDesc.appUiUrl}> {executor.appDesc.name}</a>
```

Comment:
yes, I also think so. @dongjoon-hyun Is it necessary to make a non - empty judgment?

    {
            if ({executor.state == ExecutorState.RUNNING && **executor.appDesc.appUiUrl.nonEmpty**}) {
              <a href={executor.appDesc.appUiUrl}> {executor.appDesc.name}</a>
            } else {
              {executor.appDesc.name}
            }
          }